src/Microsoft.ML.Transforms/MissingValueIndicatorTransformer.cs

```diff
@@ -427,6 +429,26 @@ private void FillValues(int srcLength, ref VBuffer<bool> dst, List<int> indices,
         }
     }
 
+    /// <summary>
+    /// <see cref="IEstimator{TTransformer}"/> for the <see cref="MissingValueIndicatorTransformer"/>.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type | Vector or scalar value of floats or doubles. |
```

Comment:
>floats or doubles [](start = 61, length = 17)

please use <xref:System.Single> and <xref:System.Double>. F# naming to totally different so we want to use .NET types.